builds.yml

```diff
@@ -19,6 +19,7 @@ buildTypes:
       - snaps
       - keyring-snaps
       - blockaid
+      - notifications
```

Comment:
If we are releasing to main, we don't need codefencing.
Also we can release the initial feature under experimental - (internal state where turned off.)

Reply:
(I am going to leave some checkboxes to make it easy for me to track remaining work)

- [ ] Remove code fences
- [ ] add experimental toggle

Reply:
Thanks for this, I'll make sure to get this ticked off!

Do we have a link to some examples of experimental features. I'll do a deep dive.